a/azure-sdk-for-python/azure-mgmt-batch_5.0.1_ubi_8.txt

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : azure-mgmt-batch
+# Version       : 5.0.1
+# Source repo   : https://github.com/Azure/azure-sdk-for-python
+# Tested on	: UBI 8.5
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vaibhav Bhadade <vaibhav.bhadade@ibm.com> 
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+PACKAGE_NAME=azure-mgmt-batch
+PACKAGE_VERSION=${1:-azure-mgmt-batch_5.0.1}
+PACKAGE_URL=${2:-https://github.com/Azure/azure-sdk-for-python}
+PACKAGE_FOLDER=azure-sdk-for-python
+
+yum install -y git  python3 python3-devel make gcc-c++ rust-toolset openssl openssl-devel libffi libffi-devel 

```

Comment:
Please check for the required installations,
I see below are the languages it required :
![image](https://user-images.githubusercontent.com/69784217/151968253-f35314f6-b6db-43db-bd43-63c8c99b5ff9.png)